src/Components/ServiceScene/LogsListScene.tsx

```diff
@@ -170,6 +180,85 @@ export class LogsListScene extends SceneObjectBase<LogsListSceneState> {
         }
       })
     );
+
+    // Subscribe to ServiceScene data changes for error handling (all visualization types)
+    const serviceScene = sceneGraph.getAncestor(this, ServiceScene);
+    this._subs.add(
+      serviceScene.subscribeToState((newState, prevState) => {
```

Comment:
I don't think this is working as intended, still seeing the empty panels unless I refresh the page

<img width="1728" height="665" alt="image" src="https://github.com/user-attachments/assets/eb92d7ea-ad35-4267-b44a-0cf0e63a7231" />

Reply:
ahhh I think it's because we're subscribing to the serviceScene state instead of the state of the query provider